recipe/meta.yaml

```diff
@@ -309,6 +311,7 @@ outputs:
         - {{ compiler('c') }}
         # Tried to use enable_language(C) to avoid needing this. It does not work.
         - {{ compiler('cxx') }}
+        - ripgrep
 
       commands:
         - pushd tests                 # [unix]
```

Comment:
it seems that the provided `dev_url` is missing the source code. 
additional investigation might be needed, however the following seems to be a better `dev_url`
https://github.com/python

https://github.com/AnacondaRecipes/python-feedstock/blob/9d61f812e93789b6a64e4a443dc6e2f9a48c563a/recipe/meta.yaml#L342

Reply:
Fixed, thanks

Reply:
I don't see any issue about https://docs.python.org/devguide/ ....

Reply:
the github URL does fit better with the `dev_url` for other feedstocks